contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -500,56 +353,6 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
             .div(WEI_PRECISION);
     }
 
-    function getDepositAmountForBorrow(
-        uint256 borrowAmount,
-        uint256 initialLoanDuration,        // duration in seconds
-        address collateralTokenAddress)     // address(0) means ETH
-        external
-        view
-        returns (uint256) // depositAmount
-    {
-        if (borrowAmount != 0) {
-            if (borrowAmount <= _underlyingBalance()) {
-                if (collateralTokenAddress == address(0)) {
-                    collateralTokenAddress = wethToken;
-                }
-                return IBZx(bZxContract).getRequiredCollateralByParams(
-                    loanParamsIds[uint256(keccak256(abi.encodePacked(
-                        collateralTokenAddress,
-                        true
-                    )))],
-                    borrowAmount
-                ).add(10); // some dust to compensate for rounding errors
-            }
-        }
-    }
-
-    function getBorrowAmountForDeposit(
```

Comment:
Breaking change for the UI

Reply:
Kevin is aware of this. I am going to keep this in mind as well.